src/plugins/npm/__tests__/npm.test.ts

```diff
@@ -533,8 +546,27 @@ describe('canary', () => {
       }
     `;
 
-    const canaryVersion = '1.2.3-canary.123.1';
-    await hooks.canary.promise(canaryVersion);
+    await hooks.canary.promise(SEMVER.patch, '');
     expect(exec.mock.calls[0][1]).toContain('lerna');
   });
+
+  // test('works when PR has "skip-release" label', async () => {
```

Comment:
How well does independent mode work with skipped releases?

Reply:
Independent mode doesn't really effect `auto version` much so the skip release label still works fine. this test was in the wrong place. Moved to auto.test.ts "canary"